docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/AveragedPerceptronWithOptions.cs

```diff
@@ -25,7 +25,7 @@ public static void Example()
             {
                 LossFunction = new SmoothedHingeLoss.Options(),
                 LearningRate = 0.1f,
-                DoLazyUpdates = false,
+                LazyUpdates = false,
                 RecencyGain = 0.1f,
                 NumberOfIterations = 10
```

Comment:
```suggestion
                MaximumNumberOfIterations = 10
```

Reply:
No, It's number of iterations.

---
In reply to: [263972500](https://github.com/dotnet/machinelearning/pull/2892#discussion_r263972500) [](ancestors = 263972500)